.github/workflows/lint.yaml

```diff
@@ -9,15 +9,18 @@ on:
       - master
 
 env:
+  GO_VERSION: 1.17.6
   NODE_VERSION: 16.13.0
 
 jobs:
   go:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Run lint
-        run: make lint/go
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@f3dc5fadcaff5d8da3574b129a58db433171b1a8 #v2.1.3
+        with:
+          go_version: ${{ env.GO_VERSION }}
```

Comment:
[nits] Is it better we specify `golangci_lint_version` expressively ?

Reply:
tbh, it's okay not to be set that one because we are using `disable-all`. it's up to you🙌

Reply:
You're right. Let me add it.